src/meta/src/controller/cluster.rs

```diff
@@ -369,6 +370,32 @@ impl ClusterController {
             .await
             .get_worker_extra_info_by_id(worker_id)
     }
+
+    pub async fn get_back_pressure(&self) -> MetaResult<GetBackPressureResponse> {
+        let nodes = self
+            .inner
+            .read()
+            .await
+            .list_active_serving_workers()
+            .await
+            .unwrap();
+        let mut back_pressure_infos: Vec<BackPressureInfo> = Vec::new();
+        for node in nodes {
+            let client = self.env.stream_client_pool().get(&node).await.unwrap();
+            let request = risingwave_pb::stream_service::GetBackPressureRequest {};
+            back_pressure_infos.extend(
+                client
+                    .get_back_pressure(request)
+                    .await
+                    .unwrap()
```

Comment:
Leave a warning instead of unwrap here. Because if there are network issues or if cn happens to be rebooting, Meta will  panic.